app/views/steps/_form.html.erb

```diff
@@ -59,11 +59,69 @@
 </div>
 
 <div class="form-group">
-  <div>
-    <p>
-      <%= step.broken_links %>
-    </p>
-  </div>
+  <% if step.broken_links && step.broken_links.count > 0 %>
+    <div class="broken-links">
+      <div class="panel-group panel-warning" id="brokenLinksAccordion" role="tablist" aria-multiselectable="true">
+        <div class="panel-heading">
+          <%= render 'shared/steps/broken-links-summary', step: step %>
+        </div>
+        <% step.broken_links.each.with_index do |link, index| %>
+          <div class="panel panel-warning">
+            <div class="panel-heading" role="tab" id="brokenLinkHeading<%= index %>">
+              <a role="button" class="collapsed" data-toggle="collapse" data-parent="#brokenLinksAccordion" href="#brokenLink<%= index %>" aria-expanded="true" aria-controls="brokenLink<%= index %>">
+                <p class="panel-title h4">
+                    <%= link['uri'] %>
```

Comment:
I think you might need to put in something here to account for long URLs, either using CSS to wrap them or truncate them (although that may not be helpful) in the code.

<img width="836" alt="screen shot 2018-08-10 at 14 41 25" src="https://user-images.githubusercontent.com/861310/43961014-c009ab80-9cab-11e8-8618-a697067294ce.png">